src/main/java/sk/freemap/gpxAnimator/Configuration.java

```diff
@@ -142,13 +144,16 @@ public Double getSpeedup() {
 	public long getTailDuration() {
 		return tailDuration;
 	}
-	
-	
+
+	public Color getTailColor() {
+		return tailColor;
+	}
+
 	public double getFps() {
 		return fps;
 	}
-	
-	
+
+
```

Comment:
There are a lot of code format changes like this one (spaces vs. tabs) in this file. Please don't mix code format changes with new features.